trade-finance-manager-ui/templates/case/facility/facility.njk

```diff
@@ -6,6 +6,8 @@
 {% import '../amendments/_macros/amendment-in-progress-bar.njk' as amendmentInProgressBar %}
 {% from "govuk/components/tabs/macro.njk" import govukTabs %}
 
+{% block pageTitle -%}Deal {{ deal.details.ukefDealId }} Facility {{ facility.ukefFacilityId }}{%- endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.